packages/arb-token-bridge-ui/src/components/TransferPanel/TokenApprovalDialog.tsx

```diff
@@ -25,7 +25,10 @@ import {
 import { TOKEN_APPROVAL_ARTICLE_LINK } from '../../constants'
 import { useChainLayers } from '../../hooks/useChainLayers'
 import { getContracts } from '../../hooks/CCTP/useCCTP'
-import { getL1GatewayAddress, getL2GatewayAddress } from '../../util/TokenUtils'
+import {
+  fetchErc20L1GatewayAddress,
+  getL2GatewayAddress
```

Comment:
they are a pair which do the same thing on L1 and L2 respectively, so change them together?